src/API/Google/AdsCampaignCriterion.php

```diff
@@ -35,6 +55,35 @@ function ( $location_id ) use ( $campaign_resource_name ) {
 			$location_ids
 		);
 	}
+	/**
+	 * Returns a set of operations to delete multiple campaign criteria.
+	 *
+	 * @param string $campaign_resource_name
+	 * @return array
+	 */
+	public function delete_operations( string $campaign_resource_name ): array {
+		$operations = [];
+
+		$results = ( new AdsCampaignCriterionQuery() )
+			->set_client( $this->client, $this->options->get_ads_id() )
+			->where( 'campaign_criterion.campaign', $campaign_resource_name )
+			->where( 'campaign_criterion.negative', 'false', '=' )
```

Comment:
When we want to delete the campaign completely, wouldn't it make sense to remove all location criteria, regardless if they are negative or not?

Reply:
Makes sense, thanks for the suggestion. Updated in 5a8c28f.